### ISSUE TITLE
Recent modifications cause a new early build failure under linux

### ISSUE THREAD

**Issue by ei-grad** [kevinbentley/Descent3#41]
```
[  0%] Building CXX object 2dlib/CMakeFiles/2dlib.dir/font.o
g++: fatal error: no input files
compilation terminated.
/bin/sh: line 1: -m32: command not found
```

**Comment by ei-grad** [comment]
Caused by https://github.com/kevinbentley/Descent3/commit/116f3d675

**Comment by Arcnor** [comment]
Indeed, `LIST(APPEND` should be `STRING(APPEND`, let's fix that with the rest of Linux fixes (there are a lot of in-flight PRs for Linux already)

**Comment by Lgt2x** [comment]
Fixed by #20
